Source/WebCore/Modules/mediastream/MediaStream.cpp

```diff
@@ -123,13 +123,17 @@ RefPtr<MediaStream> MediaStream::clone()
 {
     ALWAYS_LOG(LOGIDENTIFIER);
 
+    RefPtr document = protectedDocument();
```

Comment:
Per the style guide, we only use `protectedFoo()` functions to avoid local variables, not to initialize local variables. There is no perf / safety benefit to be calling the longer function name here.

```
RefPtr document = this->document();
```
is what we would usually use.